go-controller/pkg/ovn/pods.go

```diff
@@ -100,9 +105,6 @@ func (oc *DefaultNetworkController) deleteLogicalPort(pod *kapi.Pod, portInfo *l
 	if pod.Spec.HostNetwork {
 		return nil
 	}
-	if !util.PodScheduled(pod) {
```

Comment:
I would put this back and leverage the many iterations we have had on dealing with bugs on pod removal. ;)

Reply:
oh he removes it from here and puts it in his logic of figuring out if a pod is local to zone or not.. so he is just moving places, not moving the logic...

Reply:
That's correct